src/commands/info/about.js

```diff
@@ -13,7 +13,7 @@ module.exports = class About {
 
     client.shard.fetchClientValues("guilds.cache.size").then(results => {
       const guildCount = results.reduce((acc, guildCount) => acc + guildCount, 0);
-      const hearts = ["💖", "💝", "💓", "💗", "💘", "💟", "💕", "💞"];
+      const hearts = ["💖", "💝", "💓", "💗", "💘", "💟", "💕", "💞", "🫶"];
```

Comment:
the heart is invisible for windows 10 users

Reply:
wot? oh, okay lmao